config.toml

```diff
@@ -75,7 +75,7 @@ unsafe= true
     [Languages.fr]
     weight = 1
     languageCode = "fr-fr"
-    title = "Particule | Cloud Native Infrastructure"
+    title = "particule. | Cloud Native | GitOps | Infra as Code"
```

Comment:
Not sure about than change.
Anyway, I'd prefer `particule. | Cloud Native - GitOps - Infra as Code`

Reply:
Yes I don't mind the `|` or  `-`